### PR TITLE
[0007] update glossary

### PR DESCRIPTION
**Pull request by zengbing15**
update the description of outbound peer&inbound peer

### CODE REVIEW

File: rfcs/0007-scoring-system-and-network-security/0007-scoring-system-and-network-security.zh.md [hunk 1]
```diff
@@ -48,8 +48,8 @@ RFC 描述了客户端应该实现的打分系统和下文的几种安全策略
 * `Peer` - 网络上的其他节点
 * `PeerInfo` - 描述 Peer 信息的数据结构
 * `PeerStore` - 用于存储 PeerInfo 的组件
-* `outbound peer` - 连接由节点发起
-* `inbound peer` - 连接由 Peer 发起
+* `outbound peer` - 连接由节点主动发起
```
Comment (jjyr) [inline rfcs/0007-scoring-system-and-network-security/0007-scoring-system-and-network-security.zh.md]:
```suggestion
* `outbound peer` - 主动发起连接的节点
```

File: rfcs/0007-scoring-system-and-network-security/0007-scoring-system-and-network-security.zh.md [hunk 2]
```diff
@@ -48,8 +48,8 @@ RFC 描述了客户端应该实现的打分系统和下文的几种安全策略
...
+* `inbound peer` - 连接由节点被动发起
```
Comment (jjyr) [inline rfcs/0007-scoring-system-and-network-security/0007-scoring-system-and-network-security.zh.md]:
```suggestion
* `inbound peer` - 被动接受连接的节点
```

File: rfcs/0007-scoring-system-and-network-security/0007-scoring-system-and-network-security.md [hunk 3]
```diff
@@ -46,8 +46,8 @@ CKB client should implement the scoring system and following security strategies
 * `Peer` - Other nodes connected through the network
 * `PeerInfo` - A data struct used for describing information of `Peer`
 * `PeerStore` - A component used to store `PeerInfo`
-* `outbound peer` - describe a peer which we initiate a connection.
-* `inbound peer` - describe a peer which the peer initiates a connection.
+* `outbound peer` - describe a peer which initiatively generates a connection.
```
Comment (jjyr) [inline rfcs/0007-scoring-system-and-network-security/0007-scoring-system-and-network-security.md]: Typically we do not say `generate a connection`, `initiate a connection` is more generally used.

File: rfcs/0007-scoring-system-and-network-security/0007-scoring-system-and-network-security.md [hunk 4]
```diff
@@ -46,8 +46,8 @@ CKB client should implement the scoring system and following security strategies
...
+* `inbound peer` - describe a peer which passively generates a connection.
```
Comment (jjyr) [inline rfcs/0007-scoring-system-and-network-security/0007-scoring-system-and-network-security.md]: Inbound peer `accept a connection` not `generates` one.

Comment (zengbing15) [reply rfcs/0007-scoring-system-and-network-security/0007-scoring-system-and-network-security.md]: ok，I understand

**Pull request merged**